vendor.mod

```diff
@@ -78,37 +79,20 @@ require (
 	go.uber.org/zap v1.10.0 // indirect
 	golang.org/x/sys v0.0.0-20210823070655-63515b42dcdf // indirect
 	golang.org/x/text v0.3.4 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	google.golang.org/genproto v0.0.0-20201110150050-8816d57aaa9a // indirect
-	gopkg.in/fsnotify.v1 v1.4.7 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
-	github.com/Microsoft/go-winio => github.com/Microsoft/go-winio v0.4.15
-	github.com/akutz/gosync => github.com/akutz/gosync v0.1.0
 	github.com/containerd/containerd => github.com/containerd/containerd v1.5.2
```

Comment:
Was this replace still needed? Wondering if we can just update the indirect dependency to the latest v1.5.x release

I should note that _if_ we need to keep some of the replace rules, then it might be good to add a comment in the corresponding `require` section, and to add a comment that "the actual version is overridden in the `replace` section", similar to https://github.com/moby/buildkit/blob/390c6886f4a310af5ee0d45eaab35dc36b21f23e/go.mod#L22-L23

It's easy to overlook that there's a `replace` rule for a dependency, and people may then forget to update (or remove) them when updating dependencies.

Reply:
As the time of the PR states, the goal is to remove only some of the `replace` directives, i.e. seemingly trivial version bumps that won't affect downstream in any way. So runc and containerd are untouched for the purpose of this PR, as I cannot assume those are trivial and won't affect Moby.